iceoryx_posh/test/moduletests/test_roudi_iceoyx_roudi_memory_manager.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2021 by Robert Bosch GmbH. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "test.hpp"
+
+#include "iceoryx_posh/roudi/memory/iceoryx_roudi_memory_manager.hpp"
+
+using namespace ::testing;
+
+using iox::roudi::IceOryxRouDiMemoryManager;
+
+namespace iox
+{
+namespace test
+{
+
+/// @brief This test file verifies that the BaseClass IceoryxRouDiMemoryManager is tested
+class IceoryxRoudiMemoryManager_test : public Test
+{
+  public:
+    IceOryxRouDiMemoryManager* m_roudiMemoryManagerTest{nullptr};
+
+    void SetUp() override
+    {
+      auto config = iox::RouDiConfig_t().setDefaults();
+      m_roudiMemoryManagerTest = new IceOryxRouDiMemoryManager(config);
+    }
+
+    void TearDown() override
+    {
+  delete m_roudiMemoryManagerTest;
```

Comment:
could you please run `clang-format` to get the indentation right?

Also if you use a `unique_ptr` you don't need to delete it manually. I would suggest to use the smart pointer in the tests since it makes things easier. I always change it when possible when I see them in tests.